app/assets/stylesheets/application.scss

```diff
@@ -2,13 +2,15 @@
 @import 'variables';
 
 // Dependencies
+@import '../../../node_modules/bootstrap/scss/bootstrap';
```

Comment:
Can we use `@import 'bootstrap';` only?

Reply:
No we can't. I have to locate `bootstraps.scss` in `node_modules`.
bootstrap.scss imports all bootstrap things. I do this way because I want to use bootstrap variable in my css stuffs.